src/js/page/ui/settings.js

```diff
@@ -83,6 +89,47 @@ export default class Settings {
     this.emitter.emit('change');
   }
 
+  _onExport() {
+    this._exportRipple.animate();
+
+    const { fingerprint, multipass, pretty, ...settings } = this.getSettings();
+    const floatPrecision = Number(settings.floatPrecision);
+    const plugins = [];
+
+    for (const [name, active] of Object.entries(settings.plugins)) {
```

Comment:
Most of this block seems to be duplicated in svgo-worker. Doesn't seem good from a maintenance perspective.

Reply:
I decided to copy the code first. I can rework it a bit and take out the repeating code.

Reply:
I think that would help, yeah. Because this kind of duplication can be bad in the future.

Maybe we should just have a function to return the config with plugins somewhere in a common place.

@jakearchibald any suggestions?

Reply:
Moved the code for getting plugins into a separate utility